classes/console/console.gd

```diff
@@ -28,14 +28,13 @@ func run_command(cmd: String):
 					Singleton.log_msg("No second argument.", Singleton.LogType.ERROR)
 					return
 				match args[1]:
-					"t1":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_1.tscn"
-					"t1r2":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_2.tscn"
-					"t1r3":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_3.tscn"
-					"t1r4":
-						path = "res://scenes/levels/tutorial_1/tutorial_1_4.tscn"
+					"t1", "tutorial_1":
+						if len(args) == 2:
+							path = "res://scenes/levels/tutorial_1/tutorial_1_1.tscn"
+						else:
+							path = "res://scenes/levels/tutorial_1/tutorial_1_" + args[2] + ".tscn"
```

Comment:
Maybe check if that scene at that path actually exists, so the player can't crash the game with `w t1 somethingrandom`.

Reply:
For some reason it doesn't crash the game and I don't know why.
I will have to figure out how to check if it exists cuz I don't know how that works yet.

Reply:
ok fixed

Reply:
I realize now that there was actually already a fallback in place for it, but yours is better, I think.